lib/Chunk.js

```diff
@@ -6,23 +6,29 @@
 
 const util = require("util");
 const compareLocations = require("./compareLocations");
+const SortableSet = require("./util/SortableSet");
 let debugId = 1000;
 
-const byId = (a, b) => {
-	if(a.id < b.id) return -1;
-	if(b.id < a.id) return 1;
-	return 0;
-};
-
 class Chunk {
 
+	static sortById(a, b) {
```

Comment:
Initially I wanted to flip them, so `sortByXXX` was in `Module` and `Chunk` would use `Module.sortByXXX` to sort its Modules. 
As Module should know how to sort modules, not Chunk.
However node and circular dependencies let me down and so this is a left over of that approach, happy to make them private again, but I'd rather find a solution for the intial idea. :D